### PR TITLE
MS Entra authentication

### PR DESCRIPTION
**Pull request by williamspindox**
Thank you for taking time to contribute this pull request!
You might have already read the [contributor guide][1], but as a reminder, please make sure to:

* Sign the [contributor license agreement](https://cla.pivotal.io/sign/spring)
* Rebase your changes on the latest `main` branch and squash your commits
* Add/Update unit tests as needed
* Run a build and make sure all tests pass prior to submission

### CODE REVIEW

**Comment by tzolov** [comment]
@williamspindox thank you for the contribution
It would help to provide some context before and with the  PR?

File: document-readers/tika-reader/pom.xml
Comment (williamspindox) [inline document-readers/tika-reader/pom.xml:1]: It's not related to MS Entra, but it avoid a WARNING during build.

**Comment by williamspindox** [comment]
When can you approve my pull request? Thank you

**Comment by markpollack** [comment]
Apologies, it is a hard area for me to come up to speed and we let this slip.  I'm scheduling for the M2 release now.

**Pull request closed without merging**